extensions/narayana-lra/deployment/src/test/java/io/quarkus/narayana/lra/test/LRAOpenAPIExcludedTest.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.narayana.lra.test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class LRAOpenAPIExcludedTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest().withApplicationRoot((jar) -> jar
+            .addAsResource(new StringAsset("quarkus.lra.openapi.included=false"),
+                    "application.properties"));
+
+    @Test
+    public void testHealthServlet() {
```

Comment:
The method name requires an adjustment.

Reply:
@gsmet Thank you for the review.
Method name adjustment is done.